elixir/apps/domain/lib/domain/actors/membership_rule.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Domain.Actors.MembershipRule do
+  use Domain, :schema
+
+  @primary_key false
+  embedded_schema do
+    # `true` is a special operator which allows to select all account identities
+    field :operator, Ecto.Enum, values: ~w[true contains does_not_contain is_in is_not_in]a
```

Comment:
Just curious on the naming of the value `true`.  Was this preferred over using `all`?

Reply:
It was all and then I made it true, I kinda don't like any of those but kept true because it's in the "operator" field and reflects what generated SQL does (SELECT ... WHERE TRUE).

Reply:
Gotcha.  Makes sense.  Just figured I'd ask.